recipe/meta.yaml

```
@@ -1,18 +1,18 @@
 {% set name = "google-cloud-storage" %}
-{% set version = "2.6.0" %}
+{% set version = "2.17.0" %}
```

```suggestion
{% set version = "2.18.0" %}
```
Version `2.18.0` seems to have minor changes if you want to update to the latest

---

https://github.com/googleapis/python-storage/blob/v2.18.0/setup.py